packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchFieldProps} from '@react-types/searchfield';
+import {SearchWithin} from '../';
+import {SpectrumPickerProps} from '@react-types/select';
+import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
+import {storiesOf} from '@storybook/react';
+
+storiesOf('SearchWithin', module)
+  .add(
```

Comment:
we should move this file to CSF as well, like the chromatic stories, no reason not to
lets add some actions to the searchfield and scope picker so that the stories log them out